src/components/Toggle/Toggle.stories.mdx

```diff
@@ -0,0 +1,180 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { HStack, VStack, Grid } from "@chakra-ui/react";
+import Heading from "../Heading/Heading";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+import Toggle from "./Toggle";
+import { ToggleSizes } from "./ToggleSizes";
+
+<Meta
+  title={getCategory("Toggle")}
+  component={Toggle}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=31581%3A302",
+    },
+    jest: ["Toggle.test.tsx"],
+  }}
+  argTypes={{
+    className: { table: { disable: true } },
+  }}
+/>
+
+# Toggle
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.25.3`   |
+| Latest            | `0.25.3`   |
+
+<Description of={Toggle} />
+
+The Toggle component is used as an alternative for the Checkbox component and returns a boolean response. Like the Checkbox component, the Toggle component can display a label to the right of the toggle and helper/error text below the toggle.
+
+## Best Practices
+
+Toggle components are commonly used for “on/off” switches to perform binary actions that occur immediately after a user “flips the switch.”
+
+The Toggle component label should clarify the action being performed. Labels should be short and to the point, often three words or less.
+
+<Canvas>
+  <Story
+    name="Toggle"
+    args={{
+      labelText: "Test Label",
+      size: "lg",
+      defaultChecked: false,
+      helperText: "This is the helper text!",
+      id: "",
```

Comment:
For the purposes of Storybook, disable this `id` prop by adding it to the `argsType` prop on line 28. I was trying to toggle by clicking on the label and it wasn't working. It failed because it's set to `""` by default and so the label/input linking doesn't work.